docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Grafana Mimir version 2.1 release notes"
+menuTitle: "V2.2 release notes"
+description: "Release notes for Grafana Mimir version 2.2"
+weight: 300
+---
+
+# Grafana Mimir version 2.2 release notes
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Below we highlight the top features, enhancements and bugfixes in this release, as well as relevant callouts for those upgrading from Grafana Mimir 2.1.
+The complete list of changes is recorded in the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for ingesting out-of-order samples**: Grafana Mimir introduces new experimental support for ingesting out-of-order samples. Feature is enabled by configuring the time window since the most-recently ingested sample for the series. This can be enabled on per-tenant basis.
+
+- **Error messages**: We have overhauled the error messages reported to end user to make them easier to understand, and to include error codes that can be easily searched for. This effort will continue in next releases as well.
+
+- **Prefixed bucket store**: It is now possible to store blocks, rules and alerts into a single bucket, by configuring bucket prefix for each feature.
+
+- **Backfill API**: Grafana Mimir 2.2 introduces new API for backfill. Work on client is in progress. (Shall we drop this until client is available?)
+
+## Upgrade considerations
+
+We have updated default values and some parameters in Grafana Mimir 2.2 to give users better out-of-the-box experience:
+
+- Message size limits for gRPC messages exchanged between internal Mimir components has been increased to 100 MiB.
+
+- `-blocks-storage.bucket-store.ignore-blocks-within` parameter has been changed from `0` to `10h`. Default value of `-querier.query-store-after` has been changed from 0 to `12h`. Both changes improve query-performance for most-recent data by querying only ingesters.
```

Comment:
```suggestion
- The `-blocks-storage.bucket-store.ignore-blocks-within` parameter has been changed from `0` to `10h`. The default value of `-querier.query-store-after` has been changed from `0` to `12h`. Both changes improve query-performance for most-recent data by querying only the ingesters, rather than object storage.
```